test/dates/ranges.jl

```diff
@@ -14,7 +14,7 @@ function test_all_combos()
                 @test length(dr) == 0
                 @test isempty(dr)
                 @test first(dr) == f1
-                @test last(dr) == f1-one(l1 - f1)
+                @test last(dr) < f1
```

Comment:
I'd argue that the specific choices in how an empty range is implemented is an internal detail that we actively want to avoid testing.